main.go

```diff
@@ -17,6 +18,12 @@ func main() {
 		log.Fatal(errors.Wrap(err, "could not init logger"))
 	}
 
+	logger.Info("running app",
```

Comment:
As a variant to print this information also on `/health` page. For example https://www.uber.com/health